tests/test_models/test_losses/test_distillation_losses.py

```diff
@@ -204,3 +205,15 @@ def test_pkdloss(self):
         loss = pkd_loss(feats_S, feats_T)
         self.assertTrue(loss.numel() == 1)
         self.assertTrue(0. <= loss <= 1.)
+
+    def test_mgd_loss(self):
+        mgd_loss = MGDLoss(alpha_mgd=0.00002)
+        feats_S, feats_T = torch.rand(2, 256, 4, 4), torch.rand(2, 256, 4, 4)
+        loss = mgd_loss(feats_S, feats_T)
+        self.assertTrue(loss.numel() == 1)
+
+        num_stages = 4
+        feats_S = (torch.rand(2, 256, 4, 4) for _ in range(num_stages))
+        feats_T = (torch.rand(2, 256, 4, 4) for _ in range(num_stages))
+        loss = mgd_loss(feats_S, feats_T)
```

Comment:
`feats_S` and `feats_T` are `tuple` of `torch.Tensor` here, while the `MGDLoss` only supports `torch.Tensor` as inputs.
You can change them to `torch.rand(2, 256, 4, 4)` to resolve the error in unittest.

Reply:
There is also an error in [test_connectors](https://github.com/open-mmlab/mmrazor/actions/runs/3636395475/jobs/6136340778#step:13:165).